QualityControl/docs/Features.md

```diff
@@ -0,0 +1,33 @@
+# Features
+
+## Canvas Options via MetaData
```

Comment:
While ROOT indeed produces a canvas, majority of the users are not aware of that. I think we should keep the naming of `ROOT Object drawing options via Metadata` or something similar